package.json

```diff
@@ -54,7 +54,9 @@
     "@tauri-apps/api": "^2.0.0",
     "@tauri-apps/plugin-cli": "^2.0.0",
     "@tauri-apps/plugin-deep-link": "^2.0.0",
-    "@tauri-apps/plugin-shell": "^2.0.0",
+    "@tauri-apps/plugin-dialog": "^2.0.1",
```

Comment:
Are we using this one?

Reply:
yes, `dialog` is used for saving the file.